concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -3669,79 +3694,68 @@ jobs:
   - name: rotate-database-encryption-keys
     serial: true
     plan:
-    - aggregate:
-      - get: paas-cf
-      - get: cf-vars-store
-      - get: cf-manifest-pre-vars
-
     - task: rotate-db-encryption-key
       tags: [colocated-with-web]
       config:
         platform: linux
-        image_resource: *ruby-slim-image-resource
-        inputs:
-          - name: paas-cf
-          - name: cf-vars-store
-          - name: cf-manifest-pre-vars
-        outputs:
-          - name: modified-cf-vars-store
+        image_resource: *gov-paas-bosh-cli-v2-image-resource
+        params:
+          CREDHUB_CLIENT: credhub-admin
+          CREDHUB_SECRET: ((bosh-credhub-admin))
+          CREDHUB_CA_CERT: ((bosh-credhub-ca-cert))
+          CREDHUB_SERVER: "https://((bosh_fqdn)):8844/api"
+          DEPLOY_ENV: ((deploy_env))
         run:
           path: sh
           args:
             - -e
             - -c
             - |
-              ./paas-cf/manifests/cf-manifest/scripts/rotate-vars-store-secrets.rb \
-                --vars-store cf-vars-store/cf-vars-store.yml \
-                --manifest cf-manifest-pre-vars/cf-manifest-pre-vars.yml \
-                --passwords \
-                --rotate cc_db_encryption_key \
-                --rotate cc_db_encryption_key_id \
-                --rotate uaa_default_encryption_passphrase \
-                --rotate uaa_default_encryption_passphrase_id \
-                > modified-cf-vars-store/cf-vars-store.yml
-      on_success:
-        put: cf-vars-store
-        params:
-          file: modified-cf-vars-store/cf-vars-store.yml
+              BOSH_NS="/${DEPLOY_ENV}/${DEPLOY_ENV}"
+              credhub login
+
+              CC_DB_KEY_OLD=$(credhub get -q -n "/momo/momo/cc_db_encryption_key" | sed 's/\s//g' | tr -d '\n')
+              CC_DB_KEY_ID_OLD=$(credhub get -q -n "/momo/momo/cc_db_encryption_key_id" | sed 's/\s//g' | tr -d '\n')
+              UAA_ENCRYPTION_PASSPHRASE_OLD=$(credhub get -q -n "/momo/momo/uaa_default_encryption_passphrase" | sed 's/\s//g' | tr -d '\n')
```

Comment:
```suggestion
              UAA_ENCRYPTION_PASSPHRASE_OLD=$(credhub get -q -n "/$DEPLOY_ENV/$DEPLOY_ENV/uaa_default_encryption_passphrase" | sed 's/\s//g' | tr -d '\n')
```

etc

Reply:
Thanks Toby! I've done this, and rebased, so that I can try deploying it.